src/app/api/boosts/claim/route.ts

```diff
@@ -0,0 +1,108 @@
+import { PrismaClient } from '@prisma/client';
```

Comment:
we are no longer using prisma as it is a bottleneck, please take a look at hoq to query or insert rows with supabase sdk. 

Currently supbase sdk is not enforcing strong type, you can make db changes and insert; we will migrate to use strong types now that scalability is potentially resolved.

Take a look at: /api/webhook/treasure-box/route.ts for example of initilaizing supabase sdk and inserting